lib/ddtrace/contrib/extensions.rb

```diff
@@ -127,8 +143,6 @@ def instrument(integration_name, options = {}, &block)
             end
           end
 
-          alias_method :use, :instrument
-
```

Comment:
I'm curious about this decision. It seems really cheap/simple to keep this and support both, and it would avoid one more change that our customers need to remember to make when upgrading to 1.0.0.

This doesn't mean we don't change it everywhere in our docs, if we want to make "instrument" the recommended default; but keeping the "use" around seems like a good bang-for-the-buck.

Reply:
I think it's more semantically accurate and less likely to be confused with middleware (like Rack and Faraday do.) We may want to co-opt `use` for plugins that aren't necessarily instrumentation in the future. It's not a "must do" thing, it's a "looks better, and can do, so why not do" thing.

Reply:
The "why not" is because it's one more thing that customers will need to learn about and adapt for their 1.0 move; hence why I suggested that if the cost to us is minimal, making the customer upgrade experience better wouldn't be a bad idea.

Reply:
It's not done for no reason, it's done for reasons and greater semantic accuracy and consistent usage. 1.0 is the time to gain that consistency, if that's considered worth updating the configuration file (which they will have to change anyways.)

I'd like to hear @marcotc 's thoughts on this.

Reply:
I think we should keep the `use` alias for 1.0. We've already updated all our documentation to use `c.instrument`.